node/test/hyperbahn/advertise.js

```diff
@@ -66,4 +67,45 @@ function runTests(HyperbahnCluster) {
             assert.end();
         }
     });
+
+    HyperbahnCluster.test('can advertise using hostPortFile', {
+        size: 5
+    }, function t(cluster, assert) {
+        var bob = cluster.remotes.bob;
+        var hostPortFile = '/tmp/host-special.json';
```

Comment:
yeah... that's not going to turn out well, use some sort of standard temp file name generator please to avoid clobbering between concurrent test runs and such